spec-main/fixtures/api/beforeunload-false-prevent3.html

```diff
@@ -2,6 +2,7 @@
 <body>
 <script type="text/javascript" charset="utf-8">
   function installBeforeUnload(removeAfterNTimes) {
+    setTimeout(() => console.log('installed'))
```

Comment:
wouldn't it be better if this is added below the event listener, so that its more explicit about the order.

```
window.addEventListener('beforeunload', function handler(e) {})
setTimeout(() => console.log('installed'));
```